packages/lib/SyncTargetJoplinServer.ts

```
@@ -66,11 +66,15 @@ export default class SyncTargetJoplinServer extends BaseSyncTarget {
 	}
 
 	public async isAuthenticated() {
-		return true;
+		return this.isAuthenticatedViaJoplinServerCompatibleSession();
 	}
 
 	public static requiresPassword() {
-		return true;
+    	return false;
```

ESLint

[error] Mixed spaces and tabs.

(no-mixed-spaces-and-tabs)